src/client/cl_entity.c

```diff
@@ -427,25 +427,21 @@ void Cl_Interpolate(void) {
 			assert(mod);
 			assert(mod->bsp_inline);
 
-			ent->mins = mod->bsp_inline->mins;
-			ent->maxs = mod->bsp_inline->maxs;
+			ent->bounds = mod->bsp_inline->bounds;
 		} else {
 			angles = Vec3_Zero();
 
-			ent->mins = ent->current.mins;
-			ent->maxs = ent->current.maxs;
+			ent->bounds = ent->current.bounds;
 		}
 
 		// jdolan: Note that we use the latest snapshot origin, not the interpolated origin.
 		// This is so client side prediction can work, and the client and server are working
 		// with the same entity positions at their respective intervals.
```

Comment:
@Paril I would actually be _really_ curious to revisit this, at least for rotators, with your new tracing code. I have to wonder if we can finally do pixel-perfect prediction now by using the lerped matrices for solid models.